.github/workflows/main.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  unit:
+    name: Unit Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-elixir@v1.2.0
+        with:
+          otp-version: 21.3
+          elixir-version: 1.8.2
+      - name: Get depedencies
+        run: |
+          mix local.rebar --force
+          mix local.hex --force
+          mix deps.get
+      - name: Run unit tests
+        run: |
+          mix test
+  integration:
+    name: Integration Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-elixir@v1.2.0
+        with:
+          otp-version: 21.3
+          elixir-version: 1.8.2
+      - name: Get dependencies
+        run: |
+          mix local.rebar --force
+          mix local.hex --force
+          mix deps.get
+      - name: Run integration tests
+        run: |
+          mix test.integration
+  static:
+    name: Static Analysis
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-elixir@v1.2.0
+        with:
+          otp-version: 21.3
+          elixir-version: 1.8.2
+      - name: Retrieve cached PLT
+        uses: actions/cache@v1
+        with:
+          path: .plt
+          key: plt-${{ github.head_ref }}
+          restore-keys: |
+            plt-${{ github.base_ref }}
+            plt-master
+            plt-
```

Comment:
Same deal here as with Elsa. Make sure you generate the PLT off a merge to master.